build.sbt

```diff
@@ -56,7 +56,9 @@ lazy val buildSettings = Seq(
 lazy val dockerSettings = Seq(
   Docker / maintainer := "Snowplow Analytics Ltd. <support@snowplowanalytics.com>",
   dockerBaseImage := "snowplow/base-debian:0.2.2",
+  Docker / daemonUserUid := None, 
```

Comment:
fix the UID so that it is not unknown

Reply:
I agree this is needed!  But rather than fix it in this commit, I will include it when I merge [this issue](#164) which changes our base image to `adoptopenjdk:11-jre-hotspot-focal`, and fixes various other docker settings.

We are slowly migrating all snowplow applications to use the adoptopenjdk base image, to relieve us of the burden of maintaining our own base image.

Reply:
totally seems reasonable, I don't think it is actively breaking anything but it does make certain things challenging when connected via console haha.